src/main/java/io/jenkins/plugins/essentials/EssentialsManagementLink.java

```diff
@@ -0,0 +1,46 @@
+package io.jenkins.plugins.essentials;
+
+import hudson.Extension;
+import hudson.model.Api;
+import hudson.model.ManagementLink;
+import hudson.model.TopLevelItemDescriptor;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+import javax.annotation.CheckForNull;
+
+@ExportedBean
+@Symbol("essentials")
+@Extension
+public class EssentialsManagementLink extends ManagementLink {
+    @CheckForNull
+    @Override
+    public String getIconFileName() {
+        return "/plugin/essentials/images/essentials.svg";
```

Comment:
Does SVG actually work for `Action`s? Everywhere else I have seen PNG being used. FWIW.